### PR TITLE
TS-23288 add config for dependabot to add reviewers

### PR DESCRIPTION
**Pull request by bryanateoan**
add reviewers for dependabot

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    target-branch: main
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    labels:
+      - "dependencies"
+    open-pull-requests-limit: 0
```
Comment (cgperez) [inline .github/dependabot.yml:11]: is it correct to disable creating PRs?
> # Disable version updates for npm dependencies
    open-pull-requests-limit: 0

from https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

**Review by cgperez — CHANGES_REQUESTED** [review]
is it correct to disable creating PRs?
> # Disable version updates for npm dependencies
>    open-pull-requests-limit: 0

from https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

**Comment by bryanateoan** [comment]
> is it correct to disable creating PRs?
> 
> > # Disable version updates for npm dependencies
> > open-pull-requests-limit: 0
> 
> from https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

@cgperez This is how it was setup in `code security and analysis`. Should we include versions for this project?

**Review by cgperez — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```
Comment (cgperez) [inline .github/dependabot.yml:3]: could you add a section for `github-actions` to update the workflows external actions too

**Pull request merged**